xarray/core/accessor_str.py

```diff
@@ -104,7 +103,7 @@ def __getitem__(self, key):
         else:
             return self.get(key)
 
-    def get(self, i):
+    def get(self, i, default=""):
         """
         Extract element from indexable in each element in the array.
```

Comment:
Can we change the first line to 

```
Extract character number `i` from each string in the array
```
?

"element" is repeated currently and it makes the docstring confusing.

Reply:
Yes, that's much better. I also adjusted some of the other summary lines but didn't spend too much time on it. Hope this is not going too far.